pkg/engine/transform/transformtestfiles/k8s_slb_template.json

```diff
@@ -2298,7 +2298,8 @@
           "name": "vmLoopNode"
         },
         "dependsOn": [
-          "[concat('Microsoft.Compute/virtualMachines/', variables('masterVMNamePrefix'), copyIndex(variables('masterOffset')))]"
+          "[concat('Microsoft.Compute/virtualMachines/', variables('masterVMNamePrefix'), copyIndex(variables('masterOffset')))]",
+          "[concat('Microsoft.KeyVault/vaults/', variables('clusterKeyVaultName'))]"
```

Comment:
This demonstrates that this dependency will be removed if it's found (`k8s_slb_template.json` is the input file before we do the translation/filtering)